packages/vscode-extension/src/handlers/officeDevHandlers.ts

```diff
@@ -226,20 +220,6 @@ export async function autoOpenOfficeDevProjectHandler(): Promise<void> {
     await globalStateUpdate(GlobalKey.OpenSampleReadMe, false);
   }
   if (autoInstallDependency) {
-    if (!isManifestOnlyOfficeAddinProject(globalVariables.workspaceUri?.fsPath ?? ""))
-      void popupOfficeAddInDependenciesMessage();
```

Comment:
It seems that `popupOfficeAddInDependenciesMessage()` is not used then. Do you need to keep it?